src/constants/common.ts

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const EMPTY_ARR: any[] = [];
+
+// TODO replace EMPTY_OBJECT declarations
+// export const EMPTY_OBJECT: any = {};
+
+// TODO move NOOP here?
```

Comment:
where is noop right now?

Reply:
In `utils/function-utils`
Let's leave this comments and return to it later? Don't want to do a lot of unrelated changes here